website/docs/docs/dbt-versions/core-upgrade/05-upgrading-to-v1.10.md

```diff
@@ -31,8 +31,16 @@ New features and functionality available in dbt Core v1.10
 
 Large data sets can slow down dbt build times, making it harder for developers to test new code efficiently. The [`--sample` flag](/docs/build/sample-flag), available for the `run` and `build` commands, helps reduce build times and warehouse costs by running dbt in sample mode. It generates filtered refs and sources using time-based sampling, allowing developers to validate outputs without building entire models.
 
+### Managing changes to legacy behaviors
+
+dbt Core v1.10 introduces new flags for [managing changes to legacy behaviors](/reference/global-configs/behavior-changes). You may opt into recently introduced changes (disabled by default), or opt out of mature changes (enabled by default), by setting `True` / `False` values, respectively, for `flags` in `dbt_project.yml`.
+
+You can read more about each of these behavior changes in the following links:
+
+- (Introduced, disabled by default) [`validate_macro_args`](/reference/global-configs/behavior-changes#macro-argument-validation). If the flag is set to `True`, dbt will raise a warning if the argument `type` names you've added in your macro YAMLs don't match the argument names in your macro or if the argument types aren't valid according to the [supported types](/reference/global-configs/behavior-changes#supported-types).
+
 ## Quick hits
 
 - Provide the [`loaded_at_query`](/reference/resource-properties/freshness#loaded_at_query) property for source freshness to specify custom SQL to generate the `maxLoadedAt` time stamp on the source (versus the [built-in query](https://github.com/dbt-labs/dbt-adapters/blob/6c41bedf27063eda64375845db6ce5f7535ef6aa/dbt/include/global_project/macros/adapters/freshness.sql#L4-L16), which uses the `loaded_at_field`). You cannot define `loaded_at_query` if the `loaded_at_field` config is also provided.
```

Comment:
I think it's also worth calling this out as a feature (big or quick hit) in the upgrade guide!

Reply:
(not _just_ under **Managing changes to legacy behaviors**)

Reply:
Added to quick hits [here](https://github.com/dbt-labs/docs.getdbt.com/pull/7112/files#diff-30ad686c8233854dd1fdbe501175b4fbe5cbe04d42f4f5babf59cbb7b3a3a354R46)